### PR TITLE
Improve CLI a bit (output formatting and bug fix)

### PR DESCRIPTION
**Pull request by astorije**
First commit was started as a finishing touch of [this](https://github.com/thelounge/lounge/pull/779/files#diff-cf27c1d543e886c89cd9ac8b8aeaf05bR93), then I went a bit more crazy.

Second commit of this PR should fix https://github.com/thelounge/lounge/issues/517.
I know it is proposed to deprecate them in favor of a more generic one (`-c port=80`), but in the meantime the existing ones should work properly.

### CODE REVIEW

File: src/log.js [hunk 1]
```diff
@@ -30,3 +30,8 @@ exports.info = function() {
 exports.debug = function() {
 	console.log.apply(console, timestamp(colors.green("[DEBUG]"), arguments));
 };
+
+exports.rawInfo = function() {
+	const newArguments = timestamp(colors.blue("[INFO]"), arguments);
```
Comment (xPaw) [inline src/log.js]: Fine by me, unsure if it needed a new file though.

File: src/log.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ exports.debug = function() {
 	console.log.apply(console, timestamp(colors.green("[DEBUG]"), arguments));
 };
 
-exports.rawInfo = function() {
-	const newArguments = timestamp(colors.blue("[INFO]"), arguments);
+exports.prompt = function() {
+	const newArguments = timestamp(colors.cyan("[PROMPT]"), arguments);
 	return Array.prototype.slice.call(newArguments).join(" ");
 };
```
Comment (xPaw) [inline src/log.js]: Keep in log for now.

File: src/command-line/utils/prompt.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+"use strict";
+
+const log = require("../../log");
+const read = require("read");
+
+module.exports = (options, callback) => {
+	options.prompt = log.prompt(options.text);
```
Comment (astorije) [inline src/command-line/utils/prompt.js]: `prompt` != `text` on purpose for 2 reasons:
- We could still use `prompt` if we want to define a tailor-made CLI prompt without the `log.prompt` (we'll need to do `... = options.prompt || ...` or something though)
- `prompt({ prompt: "..." }, ...);` feels weird to me :D

Stupid? 😅

Comment (xPaw) [inline src/command-line/utils/prompt.js]: I think we can do it simpler. `log.prompt("Password:", (err, password) => {});`

**Comment by astorije** [comment]
Should be all set, @xPaw!

**Comment by astorije** [comment]
This is a small PR, but with a big enough diff: I'll wait for a second review before merging.

**Pull request merged**